source/patterns/@aws-solutions-constructs/core/lib/s3-bucket-helper.ts

```diff
@@ -166,8 +166,10 @@ export function buildS3Bucket(scope: Construct,
 
   customBucketProps = props.bucketProps ? overrideProps(customBucketProps, props.bucketProps) : customBucketProps;
 
-  const s3Bucket: s3.Bucket = new s3.Bucket(scope, _bucketId, customBucketProps);
+  // Set enforceSSL to always false to prevent duplicate policy statement
```

Comment:
I don't think we need ApplySecurePolicy anymore - we just need to make sure enforceSSL is always true unless the client explicitly says it should be false.